components/PlayerScreen.js

```diff
@@ -1,8 +1,53 @@
-import React from 'react';
-import { View } from 'react-native';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MainContain from './MainContain';
+import CardSection from './CardSection';
+import apiCall from '../helpers/API';
+import Title from './Title';
+import Treasures from './Treasures';
 
-const PlayerScreen = () => (
-  <View />
-);
+class PlayerScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      playerName: 'Tuchanka',
```

Comment:
This name sounds/reads dirty/crude. For the demo, can we use something else?

Reply:
It was a placeholder and so it will actually fetch the character name. Fun fact Tuchanka is the krogan home world in Mass effect